en/installation.rst

```diff
@@ -82,7 +82,7 @@ command:
 
 .. code-block:: bash
 
-    composer create-project --prefer-dist cakephp/app:^4.0 my_app_name
+    composer create-project --prefer-dist cakephp/app:4.* my_app_name
```

Comment:
Wouldnt `"^4.0"` work here, and keeping respective and correct sem ver operator?

Reply:
I'd normally enclose the whole argument in quotes, but unfortunately quotes don't seem to help on the Powershell. "Correctness" shouldn't matter too much here, or should it? The constraint is never stored anywhere, it will just give you the latest 4.x template, just like `^4.0` would.

Reply:
For this specific command I agree :)